mscore/albummanager.ui

```diff
@@ -196,7 +244,6 @@
   <tabstop>createNew</tabstop>
   <tabstop>load</tabstop>
   <tabstop>print</tabstop>
-  <tabstop>createScore</tabstop>
```

Comment:
why taking it out of the tabstops? I guess instead checkBoxAddPageBreak and checkBoxAddSectionBreak should get added.